perun-core/src/main/java/cz/metacentrum/perun/core/entry/TasksManagerEntry.java

```diff
@@ -90,7 +90,7 @@ public void deleteTaskResultsByIds(PerunSession sess, List<Integer> taskResultId
 			Task currTask = tasksManagerBl.getTaskById(sess, currResult.getTaskId());
 			Facility currFac = currTask.getFacility();
 			if (currFac.getId() != facility.getId()) {
-				throw new FacilityMismatchException("Tasks are from different facilities.");
+				throw new FacilityMismatchException("Tasks are from different facilities.", facility, currFac);
```

Comment:
not sure this is privilege-wise okay, i think facility admin of another facility should not get whole facility object in the error message (as he's not privileged to call getFacilityById method)